/*error can be generated during execution time*/
class TestLogicError
{
	public static void main(String args[])
	{
		int no1 =20,no2=40;
		int res =no2 - no1;
		System.out.println("Result of addition is "+res);
	}
}
